_data/tags.yml

```
@@ -58,6 +58,10 @@
   name: PrestaShop 1.7.6.x
 - slug: 1.7.6.0
   name: PrestaShop 1.7.6.0
+- slug: 1.7.7.x
+  name: PrestaShop 1.7.7.x
+- slug: 1.7.7.0
+  name: PrestaShop 1.7.7.0
```

Here are the tags used by previous beta releases:

- 1.7.3: `1.7`, `1.7.3.0`, `version`, `beta`, `releases`,  `development`
- 1.7.4: `1.7.4`, `version`, `beta`, `patch` (?), `releases`
- 1.7.5: `1.7.5`, `version`, `beta`, `minor`, `releases`
- 1.7.6: `1.7.6.x`, `version`, `beta`, `minor`, `releases`

I think the best would be to have one tag per minor release, like this:

```suggestion
- slug: 1.7.7
  name: PrestaShop 1.7.7
```